db/data/20200916151130_associate_all_ingested_entities_with_reports.rb

```diff
@@ -0,0 +1,18 @@
+class AssociateAllIngestedEntitiesWithReports < ActiveRecord::Migration[6.0]
+  def up
+    Organisation.where(service_owner: false).each do |organisation|
+      ingested_activities = Activity.where(ingested: true, organisation: organisation)
+      ingested_activities.each do |activity|
+        fund = activity.associated_fund
+        report = Report.where("description like ?", "%Historic tracker data%").find_by(fund: fund, organisation: organisation)
```

Comment:
Do we need the `like` part - it feels a bit flakey and we know there is only one report for each organisation and fund in production?

Reply:
Yeah I wasn't sure about this either - I think we can drop it given there's only one Report per fund/org on production. I was just mindful because the ticket mentions the name of the report.

Reply:
Would `where(financial_quarter: nil)` help becasuse any report the app creates will have this set, these first ones are the only ones we shoul let in without it?

Reply:
Yes good call, this should work 👍